### PR TITLE
PF-1369: delete the notebook startup script in CLI repo

### PR DESCRIPTION
**Pull request by yuhuyoyo**
the script is copied to the wsm repo now. Also the script in WSM is updated so this one is outdated.

### CODE REVIEW

File: src/main/java/bio/terra/cli/command/resource/create/GcpNotebook.java [hunk 1]
```diff
@@ -57,8 +57,6 @@
 
   @CommandLine.Option(
       names = "--post-startup-script",
-      defaultValue =
```
Comment (jaycarlton) [inline src/main/java/bio/terra/cli/command/resource/create/GcpNotebook.java:60]: Should we instead default to the URL of the one hosted in `terra-workspace-manager`?

**Review by jaycarlton — APPROVED** [review]

File: src/main/java/bio/terra/cli/command/resource/create/GcpNotebook.java (hunk 1 above)
Comment (yuhuyoyo) [reply src/main/java/bio/terra/cli/command/resource/create/GcpNotebook.java:60]: i did set the default in wsm when creating an instance. Are you worried that the CLI might be talking to an old wsm?

Comment (jaycarlton) [reply src/main/java/bio/terra/cli/command/resource/create/GcpNotebook.java:60]: No, I was worried there wouldn't be a script. This argument overrides the other one I suppose.

Maybe put somewhere in the README where the canonical location is so folks know what's there and have an idea how to write their own.

**Review by zloery — APPROVED** [review]
Just double-checked and all Broad + Verily envs have the WSM change deployed, LGTM

**Pull request merged**